src/main/java/Main.java

```diff
@@ -0,0 +1,119 @@
+import doraemonexceptions.EmptyListException;
+import doraemonexceptions.ExceedListException;
+import doraemonexceptions.InValidCommandException;
+import doraemonexceptions.IsEmptyException;
+
+import java.util.Scanner;
+public class Main {
+    public static void printLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printGreetings() {
+        printLine();
+        System.out.println("Hello! I'm Doraemon!");
+        System.out.println("What can I do for you?");
+        printLine();
+    }
+
+    public static void printMessage(Todo task, int taskNum) {
+        System.out.println("Got it. I've added this task:");
+        System.out.println("  " + task.formatTask());
+        System.out.println("Now you have " + taskNum + " tasks in the list.");
+        printLine();
+    }
+
+    public static void printMarked(Todo task) {
+        System.out.println("Nice! I've marked this task as done:");
+        System.out.println(task.formatTask());
+    }
+
+
+    public static void main(String[] args) {
+        //initializing the arrays to 100 first
+        Todo[] list = new Todo[100];
+        int taskNum = 0;
+        int pos;
+        String description;
+        String date;
+        String start;
+        String end;
+        Scanner in = new Scanner(System.in);
+        printGreetings();
+        String temp = in.nextLine();
```

Comment:
avoid using names like `temp`, use a more descriptive name like `userInput` instead

https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#use-name-to-explain